setup.py

```diff
@@ -66,6 +66,9 @@ def get_version():
             "convert2rhel = convert2rhel.initialize:run",
         ]
     },
+    install_requires=[
```

Comment:
Optional: I think it'd be useful to have a comment in the code explaining why do we have it here. To not require people delve into git history.

Reply:
Done